ocs_ci/ocs/ocp.py

```diff
@@ -530,6 +537,87 @@ def check_name_is_specified(self, resource_name=''):
                 "Resource name has to be specified in class!"
             )
 
+    def check_function_supported(self, support_var):
+        """
+        Check if the resource supports the functionality based on the
+        support_var.
+
+        Args:
+            support_var (bool): True if functionality is supported, False
+                otherwise.
+
+        Raises:
+            NotSupportedFunctionError: If support_var == False
+
+        """
+        if not support_var:
+            raise ResourceNameNotSpecifiedException(
+                "Resource name doesn't support this functionality!"
+            )
+
+    def check_phase(self, phase):
+        """
+        Check phase of resource
+
+        Args:
+            phase (str): Phase of resource object
+
+        Returns:
+            bool: True if phase of object is the same as passed one, False
+                otherwise.
+
+        Raises:
+            NotSupportedFunctionError: If resource doesn't have phase!
```

Comment:
The `NotSupportedFunctionError` exception is raised from the `check_function_supported(...)` right? Then there should probably be mentioned also `ResourceNameNotSpecifiedException` raised from `check_name_is_specified()`.